content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -74,24 +74,27 @@ Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-s
     ISSUER="sample.synsation.io"
 
     # signing key to sign the SCITT Statements
-    SIGNING_KEY="/tmp/my-signing-key.pem"
+    SIGNING_KEY="my-signing-key.pem"
 
     # File representing the signed statement to be registered
-    SIGNED_STATEMENT_FILE="/tmp/signed-statement.cbor"
+    SIGNED_STATEMENT_FILE="signed-statement.cbor"
 
     # File representing the transparent statement, which includes the signed statement and the registration receipt
-    TRANSPARENT_STATEMENT_FILE="/tmp/transparent-statement.cbor"
+    TRANSPARENT_STATEMENT_FILE="transparent-statement.cbor"
 
     # Property used to correlate a collection of statements about an artifact
     SUBJECT="my-product-id"
-    
-    # Sub Directory for SCITT scripts
-    SCRIPTS="datatrails_scitt_samples/scripts/"
 
-    # For local script execution, help Python find the modules
-    export PYTHONPATH="${PYTHONPATH}:$SCRIPTS"
+    # A command which produces a hash, eg sha256sum on linux, or shasum on macos
+    # The specific algorithm is not important for these examples
+    HASH_COMMAND=sha256sum
     ```
 
+{{< note >}}
+These defaults will place files in your current working directory. For session persistence, consider replacing the file paths with absolute paths. For example `SIGNING_KEY="$HOME/.datatrails/my-signing-key.pem"`
```

Comment:
For these samples, I'd like to keep things super simple, avoiding lots of discussion a developer should be able to infer. 
```suggestion
These defaults will place files in your current working directory.
For session persistence, consider replacing the file paths with absolute paths.
For example `SIGNING_KEY="$HOME/.datatrails/my-signing-key.pem"`
```